spec/features/visitor_can_view_categories_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Visitor can view category pages', type: :feature do
```

Comment:
Again, for the sake of consistency, we should talk about choosing one phrase together-- I used "guest user" in my tests.